src/GovUk.Education.ExploreEducationStatistics.Content.Model/User.cs

```diff
@@ -12,6 +12,8 @@ public class User
 
         public string Email { get; set; }
 
+        public DateTime? Deleted { get; set; }
```

Comment:
I don't think it would have been much more effort to add in `DeletedById` here as well. Where we've got `Deleted` or `SoftDeleted` attributes I think we've also got `DeletedById` where the value comes from `userService.GetUserId()`